synapse/tests/test_lib_storm.py

```diff
@@ -97,13 +97,34 @@ def test_storm_alltag(self):
             core.addTufoTag(node, 'foo.bar')
             core.addTufoTag(node, 'baz.faz')
 
+            # test alltag macro syntax
             node = core.eval('#foo.bar')[0]
 
             self.eq(node[1].get('inet:fqdn'), 'vertex.link')
 
             self.nn(node[1].get('#baz'))
             self.nn(node[1].get('#foo.bar'))
 
+            # test alltag macro syntax with limit
+            core.eval('[ inet:fqdn=hehe.com inet:fqdn=haha.com #lol ]')
+            self.eq(len(core.eval('#lol limit(1)')), 1)
+
+    def test_storm_limit(self):
+
```

Comment:
need a bad syntax error check here - one without a limiit valu and some with some kwarg

```
self.raises(BadSyntaxError, core.eval, 'inet:ipv4 limit()')
self.raises(BadSyntaxError, core.eval, 'inet:ipv4 limit(nodes=10)')
# int("beer") raises a ValueErrror but i don't know if the lifts would raise another error elsewhere.
self.raises(ValueError:, core.eval, 'inet:ipv4 limit("beer")')
```